### PR TITLE
Resolved Issue#4401

### PR DESCRIPTION
**Pull request by ps-19**
This pull request makes the following changes:
- Fixes Broken Links
- Resolved Issue# [4401](https://github.com/ushahidi/platform/issues/4401)
- Resolved Issue# [4409](https://github.com/ushahidi/platform/issues/4409)

Test checklist:
- [x] Tested the PR

- [x] I certify that I ran my checklist

Fixes ushahidi/platform# .
Issue #4401

Ping @ushahidi/platform
@Angamanga @ptandler @webong

[ptandler](https://github.com/ptandler)
[Angamanga](https://github.com/Angamanga)
[webong](https://github.com/webong)

Sir or Mam please accept the PR.

### CODE REVIEW

**Comment by ps-19** [comment]
@Angamanga mam please merge the PR.

**Comment by Angamanga** [comment]
@webong Could you please review

File: LICENSE.md [hunk 1]
```diff
@@ -24,20 +23,22 @@ library is linked to its license below:
...
-Map Data
-========
+# Map Data
 
-The default map data used comes from OpenStreetMap - <http://osm.org/copyright>
-The default imagery comes from MapQuest and is subject to the following terms: <http://developer.mapquest.com/web/info/terms-of-use> and <http://info.mapquest.com/terms-of-use/>
+The default map data used comes from [OpenStreetMap](http://osm.org/copyright).
+The default imagery comes from MapQuest and is subject to the following terms: 
+- [General](https://hello.mapquest.com/terms-of-use),
+- [For Developers](https://developer.mapquest.com/legal)
```
Comment (ptandler) [inline LICENSE.md:43]: For me, it's not clear, what "for developers" means in this context.

These terms-of-use start with "If you choose to use any of the MapQuest Services, you, (or if applicable, the business you represent) (“Customer”) are agreeing to abide by the conditions and conditions of this MapQuest Enterprise End User License Agreement".

I would interpret this as: if I setup an ushahidi server using MapQuest, these terms do apply. "Developer" sound for me like someone who is contributing code. Is this correct?

Comment (ps-19) [reply LICENSE.md:43]: Yes actually two different `term of use` were written in the LICENSE previously which i am trying to amend. One for `general` and additionally another for `developers`. So i have included both as written in older version on LICENSE.md file.

Comment (ps-19) [reply LICENSE.md:43]: `for developer` is term of use which developers need to understand and follow, if they are in any way using the service.

**Pull request merged**